frontend/amundsen_application/static/js/pages/TableDetailPage/WatermarkLabel/index.tsx

```diff
@@ -35,13 +33,7 @@ class WatermarkLabel extends React.Component<WatermarkLabelProps> {
 
   renderWatermarkInfo = (low: string | null, high: string | null) => {
     if (low === null && high === null) {
-      return (
-        <div className="body-2">
-          {NO_WATERMARK_LINE_1}
-          <br />
-          {NO_WATERMARK_LINE_2}
-        </div>
-      );
+      return <div className="body-2">Not Available</div>;
```

Comment:
this one by default shows:
```
Date Range
  No valid partitions found
  Unknown date range
```
which is kind confusing.

let's just update the text to "Not Available"